vfs/vfscache/item.go

```diff
@@ -439,7 +471,14 @@ func (item *Item) Open(o fs.Object) (err error) {
 
 	err = item._save()
 	if err != nil {
-		return err
+		closeErr := item.fd.Close()
+		if closeErr != nil {
+			closeErr = fs.CountError(closeErr)
+			log.Fatal(closeErr)
```

Comment:
You don't want `log.Fatal` this kills the program! Just `fs.Errorf` the error

Reply:
This was not the intention.  Will fix it accordingly.